src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/common/diff/UnifiedParser.java

```diff
@@ -187,6 +187,7 @@ else if (directive != StringUtil.charAt(diffLine,i))
 
    private boolean isNewFileLine(String nextLine)
    {
+      //TODO unsure about this check with Gary. I did change some SizeWarningWidgets to use `constants_`
```

Comment:
Do not localize these. They are for parsing output of git commands, and not shown in the UI.